build.sbt

```diff
@@ -116,29 +116,20 @@ lazy val root = (project in file("."))
 
     Compile / unmanagedSourceDirectories += (Compile / baseDirectory).value / "src" / "main" / s"scala-spark-$sparkMajorVer",
 
-    // Assembly settings
-    assembly / test := {}, // No tests in assembly
-    assemblyPackageScala / assembleArtifact := false,
-    assembly / assemblyMergeStrategy := {
-      case PathList("META-INF", xs @ _*) => MergeStrategy.discard
-      case x if x.endsWith("module-info.class") => MergeStrategy.discard
-      case x =>
-        val oldStrategy = (assembly / assemblyMergeStrategy).value
-        oldStrategy(x)
-    },
```

Comment:
I removed this because I don't think there's any need to run assembly on the root project? Unless you want to keep the ability to manual build a fat JAR